src/proxy/cpsubsystem/CPProxyManager.ts

```diff
@@ -107,8 +107,7 @@ export class CPProxyManager {
         const clientMessage = CPGroupCreateCPGroupCodec.encodeRequest(proxyName);
         return this.client.getInvocationService().invokeOnRandomTarget(clientMessage)
             .then((clientMessage) => {
-                const response = CPGroupCreateCPGroupCodec.decodeResponse(clientMessage);
-                return response.groupId;
+                return CPGroupCreateCPGroupCodec.decodeResponse(clientMessage);
```

Comment:
Minor: Can be replaced with `.then(CPGroupCreateCPGroupCodec.decodeResponse)`

Reply:
Good catch. Fixed in 1ce0e2d1eaedae13ece036ae6db2089cfa6dd745